app/views/families/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="section facilities__index">
+  <div class="container facilities__index__container">
+    <p id="notice"><%= notice %></p>
```

Comment:
I noted this in #349 but we should make a follow up task to cleanup our usage of `notice` and `altert` and convert to `flashes` as a hash. (no action needed here)